working-together/README.md

```diff
@@ -16,50 +17,59 @@ A guide for working well together.
-others' faces.
-  * Enunciate and speak loud enough so that remote members can hear you
-clearly.
-* **Introduce everyone:** Don't assume everyone has met before. Introductions 
-help break the ice and give an opportunity for voices to be heard.
-* **Pay Attention:** Be present with others during a meeting, but also engage 
-and communicate in the way you feel most comfortable.
-* **Be respectful of others' schedules:** Try to start and end meetings on time.
-* **Step up, step back:** After you speak, let others speak. If you are 
-speaking often, make sure others have had a chance to say something.
-* **Apply the Shine Theory:** Notice if someone is being cut off or ignored. Help 
-shine the light back on that contributor by repeating their idea, giving clear 
-support and credit to its source.
-* **Give credit:** Acknowledge input from others and highlight contributors. 
-Show encouragement and, if you agree with something, call it out. Ensure that 
-even the quiet voices are heard and supported.
-* **Why am I talking?:** Make sure that the thing you're about to say has not 
-already been said. If it has and has been ignored, clarify that you are 
-amplifying an under-recognized idea.
-* **One mic:** Let one person speak at a time, and let them complete their full 
-thought before responding. Don't interrupt.
-* **Clarify what you hear:** Use phrases like "What I heard you say was..." or 
-"My feedback for that is..."
-* **Speak as I:** Try to speak from the *I* perspective. Don't speak for others.
-* **Disagree:** You don't have to agree with everybody, but be aware of how you 
-frame your feedback. Criticize the idea, not the person.
-* **Everyone can participate:** Assuming someone doesn't want to participate 
-can be exclusionary. Give people the opportunity to decline an invitation.
-* **Speak up:** Follow up if team members treat others disrespectfully. If you 
-don't feel comfortable addressing someone directly, speak with your manager or Chief
-People Officer.
+- **Be kind:** Treat each other with respect.
+- **Remote matters:** Listen for and solicit input from remote members so
+  their voice is heard.
+  - Ensure the space is set up so everyone can see each
+    others' faces.
+  - Enunciate and speak loud enough so that remote members can hear you
+    clearly.
+- **Introduce everyone:** Don't assume everyone has met before. Introductions
+  help break the ice and give an opportunity for voices to be heard.
+- **Pay Attention:** Be present with others during a meeting, but also engage
+  and communicate in the way you feel most comfortable.
+- **Be respectful of others' schedules:** Try to start and end meetings on time.
+- **Step up, step back:** After you speak, let others speak. If you are
+  speaking often, make sure others have had a chance to say something.
+- **Apply the Shine Theory:** Notice if someone is being cut off or ignored. 
+Help
+  shine the light back on that contributor by repeating their idea, giving clear
```

Comment:
I can't get the "Insert a suggestion" functionality to grab all three lines, even though all are highlighted. But `Help` looks lonely all on its own line...

Reply:
Oh gosh yeah my formatter mucked this up a bit. Thanks for catching -- will fix!